exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillCursor.java

```diff
@@ -361,6 +361,9 @@ void close() {
             ExecConstants.JDBC_BATCH_QUEUE_THROTTLING_THRESHOLD );
     resultsListener = new ResultsListener(this, batchQueueThrottlingThreshold);
     currentBatchHolder = new RecordBatchLoader(client.getAllocator());
+
+    // Set Query Timeout and MaxRows
```

Comment:
Looks like `MaxRows` wasn't set here.

Reply:
Actually, we don't need it here any more. I'll fix the comment.